hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/interceptors/AutocreateTenantRequestFilter.java

```diff
@@ -46,14 +51,27 @@
 @ServerInterceptor
 public class AutocreateTenantRequestFilter implements ContainerRequestFilter {
 
+    /* list of URI chunks to which this filter should not be applied */
+    private static final List<String> uriExceptions = Arrays.asList(".*/inventory/status/?", ".*/inventory/ping/?",
+            ".*/inventory/?");
+    private static List<Pattern> uriExceptionPatterns;
+
+    static {
```

Comment:
IMHO this is OK. We don't want to recompile the regexes on every instantiation of this filter (and we don't know how many instances there will be, because that is in the hands of the container). The regex patterns are thread safe objects so having these as a constant is imho preferable to instance field.